src/foam/u2/TableView.js

```diff
@@ -46,6 +49,14 @@ foam.CLASS({
   methods: [
     function addObj(obj) {
       var e = this.start('tr')
+          .on('click', function() {
+            this.tableView.selection = obj;
```

Comment:
It's fast on my phone (although since we're binding directly to the wheel events, it doesn't work for touch except on the scrollbar).

I don't have an underpowered ARM Chromebook handy to try it on. It's acceptable on this underpowered Intel one, though it could still definitely be faster.